Source/GoBot/GoBot.swift

```diff
@@ -751,32 +748,10 @@ class GoBot: Bot, @unchecked Sendable {
             ) as? Int64 {
                 lastRxSeqFromDisk = userDefaultsValue
             }
-            lastRxSeq = max(lastRxSeqFromDB, lastRxSeqFromDisk)
+            return max(lastRxSeqFromDB, lastRxSeqFromDisk)
```

Comment:
This is interesting to me. Do we have this extra value called `lastRxSeqFromDisk` here because of the "published log"? I understand that we can't just use the database because it would be possible that some sequence numbers would be skipped as a higher sequence number would already have been retrieved via "published log"?

Reply:
Ok I answered my own question by reading Slack and the code and the answer is no, that is not the reason.